api/groups/interface.go

```diff
@@ -68,3 +68,9 @@ type ValidatorFacadeHandler interface {
 type VmValuesFacadeHandler interface {
 	ExecuteSCQuery(*data.SCQuery) (*vm.VMOutputApi, error)
 }
+
+// ActionsFacadeHandler interface defines methods that can be used from facade context variable
+type ActionsFacadeHandler interface {
+	ReloadObservers() data.NodesReloadResponse
```

Comment:
I think you should keep the pattern with (response, error) because is specific for golang programming

Reply:
I know, but I preferred to return a single DTO that contains all the info I need instead of returning 3 values + an error.